app/views/harvest_jobs/_form.html.erb

```diff
@@ -6,7 +6,7 @@
 <% unless @parser.valid_parser?(@harvest_job.environment, @version) %>
   <%= render 'parsers/error', error: @parser.error %>
 <% end %>
-<%= custom_form_with model: @harvest_job do |f| %>
+<%= custom_form_with model: @harvest_job, format: :js do |f| %>
```

Comment:
Is this the only form that need a JS response?

Reply:
I fixed the other ones ;)